www/include/core/pathway/pathway.php

```diff
@@ -131,7 +137,7 @@
     foreach ($breadcrumbData as $page => $details) {
         echo $flag;
         ?>
-        <a href="main.php?p=<?= $page . $details["opt"]; ?>" class="pathWay"><?= _($details["name"]); ?></a>
+        <a href="<?= $details['is_react'] ? "{$basePath}{$details['url']}" : "main.php?p={$page}{$details["opt"]}" ?>"<?= $details['is_react'] ? ' isreact="isreact"' : '' ?> class="pathWay"><?= _($details["name"]); ?></a>
```

Comment:
I'm not sure this is useful :
`<?= $details['is_react'] ? ' isreact="isreact"' : '' ?>`

Reply:
why?

Reply:
what is the goal of isreact="isreact" ?

Reply:
to be recognized the path of link as React for more information on how it works check here https://github.com/centreon/centreon/pull/8510/files#diff-3b51213a4742d74b47e8fa4b069c9229R301

Reply:
seems related to this : https://github.com/centreon/centreon/blob/master/www/include/core/footer/footerPart.php#L294